messages_appendix.cddl

```diff
@@ -27,8 +27,10 @@ agent-status-response = {
   ? 1: status ; status
 }
 
-status = {
-}
+status = &(
+  success: 1
+  failure: 2
+)
```

Comment:
The empty struct was intentional.  It's the place where people can add things ad-hoc or non-standard.

Reply:
Fixed